src/repl.rs

```diff
@@ -147,8 +150,9 @@ impl<F: LurkField, T: ReplTrait<F>> Repl<F, T> {
             .color_mode(rustyline::ColorMode::Enabled)
             .auto_add_history(true)
             .build();
-        let mut rl = Editor::with_config(config);
+        let mut rl = Editor::with_config(config)?;
         rl.set_helper(Some(h));
+        #[cfg(not(target_arch = "wasm32"))]
```

Comment:
How about logging an error if we are on wasm with a history path set?

Reply:
How would you set the history path to begin with on Wasm, given there is no file system?

Reply:
Oh, I am really suffering from a bad case of jet lag 🤦